pkg/processor/daemonset/daemonset.go

```diff
@@ -238,7 +238,7 @@ func processPodContainer(name string, appMeta helmify.AppMetadata, c corev1.Cont
 	}
 	c.Env = append(c.Env, corev1.EnvVar{
 		Name:  cluster.DomainEnv,
-		Value: fmt.Sprintf("{{ .Values.%s }}", cluster.DomainKey),
+		Value: fmt.Sprintf("\"{{ .Values.%s }}\"", cluster.DomainKey),
```

Comment:
Please use helm `quote` template funciton instead. See https://helm.sh/docs/chart_template_guide/functions_and_pipelines/

Reply:
thanks for the tip. This works too. I updated the PR